### ISSUE TITLE
Only Works in Editor Game View?

### ISSUE THREAD

**Issue by cweseloh** [agostontorok/slides-of-loci#3]
I tried using the sample project in Unity 2017.3.1f1.  It works great in the editor Game view.  However, when I build it (I tried both Windows and WebGL) it goes directly to the last slide.

**Comment by agostontorok** [comment]
Hi, @cweseloh I'm a little busy lately with other projects. Have you managed to figure out a solution by yourself?

**Comment by cweseloh** [comment]
There was a problem with Unity not seeing the Tags after building the project for Windows, so I had to make the slides array public and then manually add the the game objects to get it working.  Not sure if that's a Unity bug or what.  Thanks for asking!

**Comment by erdenetulga12** [comment]
@cweseloh Hey. Can you help me please. I have the same issue. Please.

**Comment by cweseloh** [comment]
Sorry - after comparing my code more carefully, I realized my previous comment wasn't accurate.  

Try replacing the Start function in the PresenterBehaviour.js file with the code below.  Then in the editor, drag each of your Slide Game Objects into the ActiveSlideView > Presenter Behaviour (Script) > Slides array.  This avoids the use of tags (and drops the support for slide sections) so it's not pretty, but it worked for me (including as a built exe outside the editor).

```
function Start() {
    // set the camera field of view to be compatible with the screen and keep the FOVhorizontal constant
    GetComponent.<Camera>().fieldOfView = GetComponent.<Camera>().fieldOfView * 16 / 9 * 1 / GetComponent.<Camera>().aspect;

    /*
	We use two tags to reference and organize slides (scenes) : Slide and Section

	Slides should be organized in sections. Sections are read ordered by how they appear in the
	hierarchy and then we iterate through slides inside each sectino and get them in the order they 
	appear in the editor window's hierarchy

	You have to create at least 1 section and 1 slide for a slideshow
    */
    slidesNum = slides.length;

    var i = 0;

    for (i = 0; i < slidesNum; i++) {
        slides[i].gameObject.GetComponent.<Camera>().enabled = false; // disable all cameras on start
    }
}
```

**Comment by erdenetulga12** [comment]
So u added function that we can manually rearrange slides. right ?

**Comment by cweseloh** [comment]
All this code change does is disable each slide's camera by default - the important part is that you need to manual add the slides to the array using the editor.  You can put them in that array in any order you want and rearrange before you build & run.

**Comment by erdenetulga12** [comment]
IT WORKED. I LOVE U SENPAI!!!
Rather interesting workaround u got there.
Only if i had thought about it much earlier.
Thanks mate.

**Comment by agostontorok** [comment]
Thanks for the workaround @cweseloh 

@erdenetulga12 I fixed with the latest version (#4 ) this editor issue with sections and slides working again. Also, the code is updated to work with Unity 2019.1.2 and license changed to MIT.
